tests/ui/needless_return.fixed

```diff
@@ -69,6 +69,27 @@ fn test_void_match(x: u32) {
     }
 }
 
+mod no_lint_if_stmt_borrows {
+    mod issue_5858 {
+        fn read_line() -> String {
+            use std::io::BufRead;
```

Comment:
Let's factor out this import at the module level, removing this one and the one in the other function.